resources/Languages/es_ES.yml

```diff
@@ -0,0 +1,134 @@
+#usos: {homelist}
+homelist: "§eTus casas: §7{homelist}"
+
+#mensajes de warp
+#usos: {warpname} {position} {world}
+warp-set: "§aHas puesto el warp §b{warpname}§a en §7{position}§a en el mundo §7{world}§a!"
+#usos: {warpname} {position} {world}
+warp-teleport: "§aFuiste teletransportado al warp §b{warpname}§a en el mundo §7{world}§a!"
+#usos: {warpname}
+warp-deleted: "§aHas eliminado con éxito el warp §7{warpname}§a!"
+warp-not-exists: "§cLa warp llamado §7{warpname}§c no existe!"
+warp-already-set: "§cEl warp seleccionado §7[§b{warpname}§7]§c ya existe en este servidor! Por favor, borre el warp y vuelva a intentarlo!"
+#usos: {warplist}
+warplist: "§eLista de warps: §7{warplist}"
+#usos: none
+no-warps-set: "§cEste servidor aún no tiene warps!"
+
+#mensajes de solicitudes de teletransportación
+tpa-send: "§aHas enviado con éxito una solicitud de tpa a §7{target}§a!"
+new-tpa: "§7{source}§e te ha enviado una solicitud de tpa, para teletransporse a ti! Utiliza §8/tpaccept {source}§e para aceptar el tpa o §8/tpdecline {source} §8para denegarlo§e!"
+new-tpa-here: "§7{source}§e te ha enviado una solicitud de tpa, para que te teletransportes a el! Utiliza §8/tpaccept {source}§e ara aceptar el tpa o §8/tpdecline {source} §8para denegarlo§e!"
+tpa-secs: "§eSu solicitud de tpa termina en §7{secs}§e segundos"
+pending-tpa-error: "§cTienes una solicitud de tpa pendiente. Por favor, espere hasta que el tpa haya sido contestado o terminado y inténtalo de nuevo luego."
+no-pending-tpa: "§cNo tienes ninguna solicitud de tpa pendiente!"
+no-self-tpa: "§cNo puedes enviarte un tpa a ti mismo!"
+tpa-secs-left: "§eLa solicitud de tpa de {source} termina en §b{secs} segundos§e!"
+player-not-found: "§cEl jugador seleccionado §7{name}§c no está en línea!"
+tpa-cancelled-by-source: "§cLa solicitud de tpa de {source} se canceló porque el jugador está desconectado."
+tpa-cancelled-by-target: "§cLa solicitud de tpa a {target} se canceló porque el jugador está desconectado."
+tpa-accepted-source: "§7{target}§a ha aceptado tu solicitud de tpa!"
+tpa-accepted-target: "§aHas aceptado con éxito la solicitud de tpa de §{source}§a!"
+tpa-declined-source: "§7{target}§c ha rechazado su solicitud de tpa!"
+tpa-declined-target: "§aHas rechazado con éxito la solicitud de tpa de §{source}§a!"
+tpa-ended: "El TpaInfo de {source} a {target} se ha terminado ahora."
+
+#retraso de spawn
+delay-tip: "§8[§eSeras teletransportado en §7{secs}§e!§8]"
+delay-stopped-by-move: "§cEl teletransporte se ha cancelado porque te has movido."
+
+#formas
+#   warps
+form-warp-menu-title: "Warps"
+form-warp-menu-content: "Por favor elecciona un warp."
+form-warp-menu-warpButton: "§b{warp}§r{line}§7clic para teletransportarse"
+#   setwarp
+form-setWarp-menu-title: "Establecer un warp"
+form-setWarp-menu-content: "Aquí puedes establecer un nuevo warp."
+form-setWarp-menu-inputNameDescription: "Nombre del nuevo warp"
+form-setWarp-menu-inputPermDescription: "Permiso del nuevo warp (opcional)"
```

Comment:
In theSpawn v1.7.0-dev (and newer) this part won't work. 
Please use:
```yaml
form-setWarp-menu-togglePermDescription: "Should the new warp have a permission? (optional)"
form-setWarp-menu-inputIconPathDescription: "Icon-path of the new warp (optional)"
```
[line 88 and 89](https://github.com/supercrafter333/theSpawn/commit/ef17d314a07d6b10b638fdbe57c90be2da6897cb#diff-0b934b18ed444bc07da12626e22dc539ba047e86c8b8a05c987df1bfb3926a28R88-R89)